ansible/roles_ocp_workloads/bookbag/defaults/main.yaml

```diff
@@ -0,0 +1,9 @@
+---
+become_override: false
+silent: false
+
+#bookbag_git_repo: #required
+#bookbag_git_version: #optional
+bookbag_namespace: "{{ guid }}-bookbag"
+#bookbag_openshift_token: #optional
```

Comment:
I guess these are for non OpenShift configs?

Reply:
Exactly.